giskard_vision/landmark_detection/tests/performance.py

```diff
@@ -74,6 +74,12 @@ class NMEMean(Metric):
     def definition(prediction_result: PredictionResult, marks: np.ndarray):
         return np.nanmean(NMEs.get(prediction_result, marks))
 
+    @staticmethod
+    def rank_data(prediction_result: PredictionResult, marks: np.ndarray):
+        predictions = NMEs.get(prediction_result, marks)
+        indexes = sorted(range(len(predictions)), key=predictions.__getitem__)[::-1]
```

Comment:
no need to copy with `::-1` here, le's just use `indexes.reverse()` (careful: it does it inplace)